numpy/core/include/numpy/ndarraytypes.h

```diff
@@ -652,6 +650,27 @@ typedef struct _arr_descr {
         PyObject *shape;       /* a tuple */
 } PyArray_ArrayDescr;
 
+/*
+ * Memory handler structure for array data.
+ */
+typedef void *(PyDataMem_AllocFunc)(size_t size);
+typedef void *(PyDataMem_ZeroedAllocFunc)(size_t nelems, size_t elsize);
+typedef void (PyDataMem_FreeFunc)(void *ptr, size_t size);
+typedef void *(PyDataMem_ReallocFunc)(void *ptr, size_t size);
+typedef void *(PyDataMem_CopyFunc)(void *dst, const void *src, size_t size);
+
+typedef struct {
+    char name[128];  /* multiple of 64 to keep the struct unaligned */
```

Comment:
```suggestion
    char name[128];  /* multiple of 64 to keep the struct aligned */
```
Fixed size, wondering if I should do that for ArrayMethod?